marionette_todomvc/js/TodoMVC.Todos.js

```diff
@@ -18,6 +18,40 @@ var TodoMVC = TodoMVC || {};
 			Backbone.Model.prototype.initialize.call(this);
 		},
 
+		handleAction: function (action) {
+			if (action.id !== this.id) {
+				return;
+			}
+
+			switch (action.type) {
+				case 'EDIT_TODO':
+					this.set('text', action.text);
+					return;
+
+				case 'COMPLETE_TODO':
+					this.set('completed', !this.get('completed'));
+					return;
+
+				case 'DELETE_TODO':
+					this.destroy();
+			}
+		},
+
+		createAction: function (eventName) {
+			var action = this.toJSON();
+			switch (eventName) {
+				case 'change:text':
+					action.type = 'EDIT_TODO';
+					return action;
+
+				case 'change:completed':
+					action.type = 'COMPLETE_TODO'
+					return action;
+			}
+
+			return;
+		},
+
```

Comment:
Any model attached to a view will have its `handleAction` method called when redux dispatches an action, and its `createAction` method called when this model triggers any event. The same is true for collections below.